ironfish/src/rpc/routes/wallet/buildTransaction.test.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { RawTransactionSerde } from '../../../primitives/rawTransaction'
+import { useAccountFixture } from '../../../testUtilities'
+import { createRawTransaction } from '../../../testUtilities/helpers/transaction'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+
+describe('Route wallet/buildTransaction', () => {
```

Comment:
would another test where we take the output of this rpc endpoint and use it to post a transaction be beneficial?

Reply:
Couldn't hurt! We have a test that does that with the `BuildTransaction` worker job, but additional test coverage for the RPCs isn't a bad thing